docs/csharp/language-reference/compiler-messages/cs8802.md

```diff
@@ -0,0 +1,49 @@
+---
+description: "Compiler Error CS8802"
+title: "Compiler Error CS8802"
+ms.date: 7/01/2024
+f1_keywords:
+  - "CS8802"
+helpviewer_keywords:
+  - "CS8802"
+---
+# Compiler Error CS8802
+
+Only one compilation unit can have top-level statements.
+
+This error indicates that there are two or more [top-level statements](../../fundamentals/program-structure/top-level-statements.md) in a single compilation unit (single solution, project or a single group of files compiled into a single binary file).
```

Comment:
I don't think "single solution" is right.

Reply:
I agree. A solution can contain multiple console applications:

```suggestion
This error indicates that there are two or more [top-level statements](../../fundamentals/program-structure/top-level-statements.md) in a single compilation unit (single project or a single group of files compiled into a single binary file).
```